dataprofiler/tests/profilers/test_float_column_profile.py

```diff
@@ -211,6 +211,13 @@ def test_profiled_precision(self):
                 msg=f"Errored for: {sample[0]}",
             )
 
+        # Validate categorical series with trailing zeros supported
+        categorical_series = pd.Series(
+            [202209, 202210, 202211], dtype="category"
+        ).apply(str)
+        float_profiler = FloatColumn("Name")
+        float_profiler.update(categorical_series)
```

Comment:
testing this locally so may have some additional comments, but initial thought is that there should definitely be some `assert` statements here to validate this is actually working as intended post-change @SchadtJ @scottiegarcia